RunningOrder/MainView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MainView.swift
+//  RunningOrder
+//
+//  Created by Clément Nonn on 23/06/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+struct MainView: View {
```

Comment:
There is already another struct with the same name

Reply:
The renaming seems to have duplicated the file 😅 I will remove the old one